prod-aws/kafka-shared-msk/energy-budget-plan/energy-budget-plan.tf

```diff
@@ -1,105 +0,0 @@
-resource "kafka_topic" "eqdb_loader" {
```

Comment:
@oliverpauffley : you don't need the data in these topics any more, right?

Reply:
Yep! namespace is gone! Thanks for checking

Reply:
We'll need to do the cleanup manually, so if you don't mind, I'll take over this PR, as we'll need to:
1. first remove only the files with the topics, not the entire module - so that we can use tf to do the deletions
2. only after this, remove the whole module

Reply:
The cleanup is done, so we can merge this.